SingularityService/src/main/java/com/hubspot/singularity/SingularityLeaderController.java

```diff
@@ -141,8 +141,8 @@ public void isLeader() {
       master = true;
       try {
         if (!isTestMode()) {
-          scheduler.start();
           statePoller.wake();
+          scheduler.start();
```

Comment:
Small change, but I'm curious: why?

Reply:
If we call the state poller first we get instant feedback in the UI about who is leader versus having to wait until after startup is already finished to know which host gained leadership